source/puppetdb/index.markdown

```diff
@@ -8,6 +8,7 @@ PuppetDB is the fast, scalable, and reliable data warehouse for Puppet. It cache
 
 ## Current Versions
 
+* [PuppetDB 3.0](./3.1)
```

Comment:
Missed the link text; still says 3.0. Also, this should say which Puppet version families are compatible, like the ones below.